contrib/testgen/gen_key_io_test_vectors.py

```
@@ -83,7 +80,8 @@
     suffix = bytearray(template[2])
     dst_prefix = bytearray(template[4])
     dst_suffix = bytearray(template[5])
-    rv = b58encode_chk(prefix + payload + suffix)
+    assert len(prefix) == 1
```

Ruff (0.12.2)

Use of `assert` detected

(S101)